changelog/fragments/fix-scorecard-config-example.yaml

```diff
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      Remove invalid entries in the scorecard config.yaml example.
```

Comment:
The only change for end users is in the docs. So, we do not need to add a fragment in the scenario, 
So, just remove this one and then, otherwise it is fine.

Reply:
removed it, since this was only a test data change you are right I think, it wasn't needed.